### PR TITLE
Fix/details severity desc

### PR DESCRIPTION
**Pull request by dorianjp**
# Summary | Résumé

Trello
https://trello.com/c/mCqGJL7E/817-fix-the-guidance-row-in-alert-history-to-reflect-the-notification-message-selected
https://trello.com/c/p4t2Wxjl/823-terms-of-use-html-title-different-the-h1

* Fixed severity descriptions from level x -> verbal descriptions
* Fixed titles on privacy and terms of use pages

# Test instructions | Instructions pour tester la modification

Alert history -> details:
See description of guidance:
image

### CODE REVIEW

File: outbreaks/models.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 from django.db import models
 from profiles.models import HealthcareUser
 from register.models import Location
 
 
-SEVERITY = [("1", "Level 1"), ("2", "Level 2"), ("3", "Level 3")]
+SEVERITY = [
+    ("1", _("Isolate and get tested")),
+    ("2", _("Self-monitor for 14 days")),
+    ("3", _("Deprecated Severity")),
```
Comment (dorianjp) [inline outbreaks/models.py:11]: left this in to not break test data

**Comment by dorianjp** [comment]
[Fix the Guidance row in alert history to reflect the notification message selected](https://trello.com/c/mCqGJL7E/817-fix-the-guidance-row-in-alert-history-to-reflect-the-notification-message-selected)

**Comment by dorianjp** [comment]
[Terms of Use HTML title different the H1](https://trello.com/c/p4t2Wxjl/823-terms-of-use-html-title-different-the-h1)

File: outbreaks/forms.py [hunk 2]
```diff
@@ -23,10 +23,7 @@ def end_date_shift_for_view(dttm):
     ("1", _("Isolate and get tested")),
     ("2", _("Self-monitor for 14 days")),
 ]
-severity_help = [
-    ("1", _("Isolate and get tested")),
-    ("2", _("Self-monitor for 14 days")),
-]
```
Comment (dsamojlenko) [inline outbreaks/forms.py:29]: I pushed this commit - this help text array isn't used - it's defined further down.

**Review by dsamojlenko — APPROVED** [review]
I pushed one small semi-related commit

**Pull request merged**